rules/REQUEST-903.9003-NEXTCLOUD-EXCLUSION-RULES.conf

```diff
@@ -262,6 +262,17 @@ SecRule REQUEST_FILENAME "@contains /remote.php/dav/addressbooks/" \
     nolog,\
     setvar:'tx.allowed_request_content_type=%{tx.allowed_request_content_type}|text/vcard'"
 
+# Allow modifying contacts via the web interface
+SecRule REQUEST_METHOD "@streq PUT" \
+    "id:9003321,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    chain"
+    SecRule REQUEST_FILENAME "@contains /remote.php/dav/addressbooks/" \
+        "t:none,\
+        ctl:ruleRemoveById=200002"
 
```

Comment:
Just noticed that there should be additional newline here to end the "section" to be consistent with the rest of this file.